src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -23,21 +24,59 @@
     <PackageReference Include="System.Runtime.Serialization.Json" Version="4.3.0" />
     <PackageReference Include="System.Security.Cryptography.Primitives" Version="4.3.0" />
     <PackageReference Include="System.Threading.Channels" Version="4.5.0" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
     <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
     <PackageReference Include="Utf8Json" Version="1.3.7" />
+    <!-- <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.0-preview.4.20202.18" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="5.0.0-preview.4.20202.18" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0-preview.4.20202.18" />
+    <PackageReference Include="Microsoft.Extensions.Logging.EventSource" Version="5.0.0-preview.4.20202.18" /> -->
     <!-- following two dependencies must point to version 4.3.0 or newer to make it possible to publish MicroBenchmarks.csproj -->
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="5.0.0-preview.4.20214.2" />
     <PackageReference Include="Microsoft.Win32.Primitives" Version="4.3.0" />
     <PackageReference Include="System.Net.Primitives" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\AspNetCoreBenchmarkAttribute.cs">
+      <Link>Shared\AspNetCoreBenchmarkAttribute.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\DefaultCoreConfig.cs">
+      <Link>Shared\DefaultCoreConfig.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\DefaultCoreDebugConfig.cs">
+      <Link>Shared\DefaultCoreDebugConfig.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\DefaultCorePerfLabConfig.cs">
+      <Link>Shared\DefaultCorePerfLabConfig.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\DefaultCoreProfileConfig.cs">
+      <Link>Shared\DefaultCoreProfileConfig.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\DefaultCoreValidationConfig.cs">
+      <Link>Shared\DefaultCoreValidationConfig.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\ParameterizedJobConfigAttribute.cs">
+      <Link>Shared\ParameterizedJobConfigAttribute.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\ParamsDisplayInfoColumn.cs">
+      <Link>Shared\ParamsDisplayInfoColumn.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\Program.cs">
+      <Link>Shared\Program.cs</Link>
+    </Compile>
+    <Compile Include="C:\CodeHub\performance\src\benchmarks\micro\libraries\Internal\ServiceProviderMode.cs">
+      <Link>Internal\ServiceProviderMode.cs</Link>
+    </Compile> -->
+  </ItemGroup>
+
```

Comment:
the referenced files were related to custom configs created by ASP.NET, here we are using our own config: https://github.com/dotnet/performance/blob/master/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs

so I think that you can just remove it

```suggestion
```